config.md

```diff
@@ -119,11 +119,11 @@ For Windows, see [mountvol][mountvol] and [SetVolumeMountPoint][set-volume-mount
 
 ## <a name="configProcess" />Process
 
-**`process`** (object, OPTIONAL) specifies the container process.
+**`process`** (object, OPTIONAL) specifies a process to run inside the container.
```

Comment:
IMO "the container process" has a very different connotation here than simply "a process to run", especially in the context of a PID namespace (which can only exist as long as "the" container process does), so this change doesn't make a lot of sense to me.